projects/fusion-ui/components/table/v3/stories/table.story-holder.component/table.story-holder.component.component.ts

```diff
@@ -15,6 +15,19 @@ import {isNullOrUndefined} from '@ironsource/fusion-ui/utils';
         (rowModelChange)="onRowModelChange($event)"
         (scrollDown)="onscrollDown()"
     ></fusion-table>`,
+    styles: [
```

Comment:
Here example of how consumers can set CSS properties for div element in the table cell.
in this story it "bage" "US" and "ROW".
Or use a dynamic component. for this cell